src/class-elements/private-field-super-access-throws.case

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2019 Caio Lima. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: Acessing private field from super shoudl throw an error
+info: |
+  Updated Productions
+
+  CallExpression[Yield, Await]:
+    CoverCallExpressionAndAsyncArrowHead[?Yield, ?Await]
+    SuperCall[?Yield, ?Await]
+    CallExpression[?Yield, ?Await]Arguments[?Yield, ?Await]
+    CallExpression[?Yield, ?Await][Expression[+In, ?Yield, ?Await]]
+    CallExpression[?Yield, ?Await].IdentifierName
+    CallExpression[?Yield, ?Await]TemplateLiteral[?Yield, ?Await]
+    CallExpression[?Yield, ?Await].PrivateName
+template: default
+features: [class-fields-private, class-fields-public]
+---*/
+
+//- elements
+#m = function() { return 'test262'; };
+
+Child = class extends C {
+  access() {
+    return super.#m;
+  }
+
+  method() {
+    return super.#m();
+  }
+}
+
+//- assertions
+assert.throws(TypeError, function() {
+  (new (new C()).Child).method();
```

Comment:
total nitpicking, and please disregard this but... `new new C().Child().method()` works too

Reply:
actually `(new (new C()).Child()).method();` is more consistent (adding parentheses to Child).